client/js/userTaskView.js

```diff
@@ -0,0 +1,193 @@
+document.addEventListener('DOMContentLoaded', function() {
+    const data = [
+        { col1: '1', col2: 'Create design', col3: '2023-12-13 12:00', col4: 'We need a design that we can discuss around...', col5: '2', col6: 'Design, CoOp', col7: 'Done', user: 'Liber09' },
+        { col1: '2', col2: 'Create first component', col3: '2023-12-18 12:00', col4: 'Create the first component used in the design', col5: '3', col6: 'Implement', col7: 'Ready for test', user: 'Liber09' },
+        { col1: '3', col2: 'Create components', col3: '2023-12-18 12:00', col4: 'Create components used in the design', col5: '3', col6: 'Implement', col7: 'Code review', user: 'User 1' },
+        { col1: '4', col2: 'Apply design', col3: '2023-12-18 12:00', col4: 'Implement the design', col5: '3', col6: 'Design', col7: 'In progress', user: 'User 2' },
+        { col1: '5', col2: 'Load content', col3: '2023-12-20 12:00', col4: 'Load content', col5: '3', col6: 'Design', col7: 'Blocked', user: 'Liber09' },
+        { col1: '6', col2: 'Deploy', col3: '2023-12-24 12:00', col4: 'Deploy everything', col5: '3', col6: 'Deploy', col7: 'ToDo', user: '' },
+    ];
+    const users = ['Liber09', 'User 1', 'User 2', 'User 3'];
+
+    const tbody = document.querySelector('tbody');
+    const table = document.getElementById('tasksTable');
+    const userList = document.getElementById('userList');
+    let selectedUser = null;
+
+    function populateUserList() {
+        // Clear existing user list
+        userList.innerHTML = '';
+
+        // Add "All Users" option
+        const allUsersOption = document.createElement('li');
+        const allUsersAnchor = document.createElement('a');
```

Comment:
The createElements could be global like this if i remember correct: `const createLiElement = () => document.createElement('li')`, and write `createLiElement();` instead of writing `document.createElement('li'`) or similar for each line where you need it. The code will also be easier to change if you would like to use another element in the future.